web/blog/public-roadmap-and-open-rfcs/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Public Roadmap and Open RFCs
+description: Open Source needs Open Governance
+authors: mavam
+draft: true
+date: 2022-09-07
+tags: [roadmap, github, open-source]
+---
+
+We are happy to announce that we have published [our technical roadmap][roadmap]
+along with an [RFC process][rfc] to actively participate in shaping upcoming
+topics. This blog post explains why and how we did it.
+
+[roadmap]: https://vast.io/roadmap
+[rfc]: /docs/develop-vast/contributing/rfc
+
+<!--truncate-->
+
+As a community-first open-source project, we constantly strive for increasing
+transparency. Our long-term goal is establishing a fully open governance model.
+This will allow for a clear delineation of the open-source project and unbiased
+commercial offerings that we, the engineering team behind VAST at
+[Tenzir](https://tenzir.com), provide on top. Until we have bootstrapped
+ourselves and an active community, we aim for the right balance between open
+and closed.
+
+One step in the direction of open is publishing our [roadmap][roadmap] and
+enabling the community to participate in the planning through an [Request For
+Comments (RFC)][rfc] process.
+
+## Public Roadmap
+
+In the process of opening the roadmap, we had to answer several questions:
+
+1. **Audience**: should the content be for users only? What about developers?
```

Comment:
```suggestion
1. **Audience**: Should the content be for users only? What about developers?
```

Reply:
I usually start lowercase after a colon, so would leave this to the author's personal style.

Reply:
Grammarly says that American English has no fixed rule on this, so whatever. (Just shows again that British English is the strictly superior language.)

> In British English, the first letter after a colon is capitalized only if it's a proper noun or an acronym; in American English, the first word after a colon is sometimes capitalized if it begins a complete sentence.

Anyways, I don't feel too strongly about this, but then at least make it consistent and make (3) lowercase after the colon as well, which is why I noticed this in the first place.

Reply:
Touché!